python/cuda_cccl/tests/cooperative/examples/block/reduce.py

```diff
@@ -9,12 +9,11 @@
 import numba
 import numpy as np
 from numba import cuda
-from pynvjitlink import patch
 
 import cuda.cccl.cooperative.experimental as coop
 
-patch.patch_numba_linker(lto=True)
 numba.config.CUDA_LOW_OCCUPANCY_WARNINGS = 0
+numba.config.CUDA_ENABLE_PYNVJITLINK = 1
```

Comment:
This can probably be eliminated if you have a lower bound on `numba-cuda`. Same in the other test files.
```suggestion
```

Reply:
Nice, I removed all instances of these. :crossed_fingers: